rancid-tomatillos/cypress/e2e/card_container.cy.js

```diff
@@ -26,6 +27,35 @@ describe("CardContainer Test Suite", () => {
       .get(".allTitles").should("exist")
       .get(".allRatings").should("exist")
   })
+
+  it("should be able to click on the 'SORT BY RATING' button and see movies displayed in order from highest to lowest rating", () => {
+    cy
+      .get(".allTitles").then(title => {
+      expect(title[0]).to.contain.text("Money Plane")
+      })
+      .get(".center").contains("SORT BY RATING")
+      .click()
+      .get(".allTitles").then(title => {
+      expect(title[0]).to.contain.text("Cats & Dogs 3: Paws Unite")
+      })
+  })
+
+  it("should be able to click on the 'SORT BY RATING' while moves are sorted from highest-to-lowest rating and change to lowest-to-highest rating", () => {
```

Comment:
Great job replicating the whole user flow with this! In later iterations I'd be curious to know if there is a way to dry up this testing a bit. Maybe I'll do some research on that later. Thank you!